docs/source/tutorials/epics/single-PV.rst

```diff
@@ -6,33 +6,98 @@ In this tutorial we will read, write, and monitor an EPICS PV in ophyd.
 Set up for tutorial
 -------------------
 
-For this tutorial you will need an EPICS IOC serving a PV that you can connect
-to.  We suggest starting a simple test IOC using `caproto`_, as follows. If
-you already have another IOC handy, you may use that instead.
+Before you begin, install ``ophyd``, ``pyepics``, ``bluesky``, and ``caproto``,
+following the :doc:`install`.
 
-Install caproto.
+We'll start this simulated hardware that implements a `random walk`_. It has
+just two PVs. One is a tunable parameter, ``random_walk:dt``, the time between
```

Comment:
Observation: So, technically, it's not a "single-PV" tutorial?

Reply:
Hm. I meant it in the sense that we'll be using `EpicsSignal` and `EpicsSignalRO` to connect to bare PVs, as opposed to grouping them into ophyd Devices.

Reply:
Right.  This follows a basic learning trajectory, use `EpicsSignal` (et al.) for PV connection, then progress to `Device` in next lesson.

Reply:
I guess to me more precise I should title this “Single EPICS PVs” instead of “A Single EPICS PV”.

Reply:
Done!